opam

```diff
@@ -21,12 +21,13 @@ build-test: [
   "ocamlbuild" "-classic-display" "-use-ocamlfind" "src_test/test_reason.byte" "--"
 ]
 depends: [
-  "easy-format"  {>= "1.2.0"}
-  "ocamlfind"    {build}
-  "utop"         {>= "1.17"}
-  "menhir"       {>= "20160303"}
-  "merlin"       {>= "2.5.0"}
-  "merlin-extend"{>= "0.3"}
+  "easy-format"   {>= "1.2.0"}
```

Comment:
I think if you rebase, you'll see we vendored `easy-format`. Just a heads up that when resolving merge conflict we should remove the easy-format dep